Xamarin.Forms.Core/VisualElementExtensions.cs

```diff
@@ -5,12 +5,12 @@ namespace Xamarin.Forms
 {
 	public static class VisualElementExtensions
```

Comment:
please rename to BindableObjectExtensions

Reply:
good catch